src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -12,7 +12,8 @@ object MockWorldState {
 case class MockWorldState(
   accounts: Map[Address, Account] = Map(),
   codeRepo: Map[Address, ByteString] = Map(),
-  storages: Map[Address, MockStorage] = Map()
+  storages: Map[Address, MockStorage] = Map(),
+  headers: Map[BigInt, ByteString] = Map()
```

Comment:
yes, true
What I want to have here is both cases when current block (CB) is actually inside blockchain:
B0 B1 B2 CB B3 ...
and case when current block is at the end of block chain
B0 B1 B2 B3 ... CB
What you propose does not allow second case